### DISCUSSION TITLE
Help with macro hot-bar needed

### DISCUSSION THREAD

**Discussion by 666panda666** [JohanFalt/Foundry_WoD20#887] [Q&A]
Hi, i woul'd like to know if is it possible to create a macro in the hotbar to roll an actor iter, eg. a discipline or a weapon attack. 
or to toggle on/off of an item on a character scheet?
if it is possible could you please give me a little hint in how to do it? 

![attack](https://github.com/user-attachments/assets/1a942639-c960-461b-aef9-8fee3a086d53)
Like in these screenshot: click on the button number 1 and open the "roll attack window"

![toggle](https://github.com/user-attachments/assets/8c64366a-3f9c-4a7b-ba38-9a812b1859f0)
Or in this screenshot click on toolbar to activate a macro that toggle on/off the item on the character sheet

Tank you very much for you precius work!

**Comment by JohanFalt** [comment]
No with how things work at the moment you can't access the sheet in such way.
